pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go

```diff
@@ -34,10 +35,13 @@ func (w *auditLogAnalyzer) CollectData(ctx context.Context, storageDir string, b
 		return nil, nil, err
 	}
 
-	auditLogSummary, auditEvents, err := intervalsFromAuditLogs(ctx, kubeClient, beginning, end)
-	w.auditLogSummary = auditLogSummary
+	w.summaryAnalyzer = &summaryAnalyzer{}
+	err = ProcessClusterEvents(ctx, kubeClient, &beginning, &end, w.summaryAnalyzer)
+	if err != nil {
+		return nil, nil, err
+	}
 
-	return auditEvents, nil, err
+	return monitorapi.Intervals{}, nil, err
```

Comment:
Drive by. This looks like a bug. It should be possible to produce intervals from the audit log that can be used by other tests.